framework/configuration/rest-resource.rst

```diff
@@ -0,0 +1,203 @@
+REST Resource
```

Comment:
For actions we have an ActionResource interface. Maybe this could be mention somewhere.